src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumesInMemoryRepository.cs

```diff
@@ -109,7 +123,26 @@ public Resume Add(Resume resume)
             return resume;
         }
 
-        public void Update(int id, Resume updateObject)
+        public Resume Update(Resume updateObject)
+        {
+            var resume = Resumes.Find(x => x.Id == updateObject.Id);
+
+            if (resume == null)
+            {
+                return resume;
+            }
+
+            var skills = resume.Skills.ToList();
+            foreach (var skill in updateObject.Skills)
+            {
+                skills.Add(skill);
+            }
+
+            resume.Skills = skills;
+            return resume;
```

Comment:
this implementation should be updateResumeSkills method, because it is specific for skills update, the update method should update all other fields